cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -125,9 +132,16 @@ data ClientCommand
     !FilePath
     !FilePath
   | SubmitTx
-    { stTopology :: TopologyInfo
-    , stTx       :: FilePath
-    , stCommon   :: CommonCLI
+    { stTopology      :: TopologyInfo
```

Comment:
These need to be removed because they are partial record accessors.

Reply:
There is a balance to be had, because legibility is also important.

A similar case was discussed in: https://github.com/input-output-hk/ouroboros-network/pull/838#discussion_r307967852

Reply:
Done!